lib/commentOnOpenedPullRequest.js

```diff
@@ -93,6 +102,23 @@ commentOnOpenedPullRequest._askAboutChanges = function (files, baseBranch) {
     return true;
 };
 
+commentOnOpenedPullRequest._askAboutContributors = function (userName, repositorySettings) {
+    if (!defined(repositorySettings.contributorsUrl)) {
+        return Promise.resolve(false);
+    }
+
+    return requestPromise.get({
+        url: repositorySettings.contributorsUrl,
+        headers: repositorySettings.headers,
+        json: true
+    })
+    .then(function (response) {
+        var contributorsFileContent = Buffer.from(response.content, 'base64').toString();
+        var regex = new RegExp('\\[.+\\]\\(https:\\/\\/github.com\\/' + userName + '\\)');
```

Comment:
I think we can just look for their username, it doesn't need to necessarily match this format.The simpler it is, the less it may need to change in the future.